cluster-scope/overlays/prod/moc/infra/kustomization.yaml

```diff
@@ -16,11 +16,11 @@ resources:
   - ../../../../base/core/namespaces/openshift-nmstate
   - ../../../../base/core/namespaces/opf-alertreceiver
   - ../../../../base/nmstate.io/nmstates/nmstate
-  - ../../../../base/operator.openshift.io/ingresscontrollers/default
```

Comment:
I don't think this should be deleted.

Reply:
woops, I intended to delete the cert, not the controller. Thanks!